core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/SchemaElementDefinitionValidator.java

```diff
@@ -168,10 +166,10 @@ protected ValidationResult validateFunctionArgumentTypes(
         return result;
     }
 
-    private ValidationResult validateDirection(final SchemaElementDefinition elementDef) {
+    protected ValidationResult validateDirection(final SchemaElementDefinition elementDef) {
```

Comment:
Why is this now protected? There are no children of the `SchemaElementDefinitionValidator` class. I also see other methods in this class which could be private.

Reply:
Because it needs to be called in the Test, a few of the other functions are the same for this reason I think. It just makes the test cleaner with less mocking if I can call the method directly

Reply:
Not exactly the best practice though, but I had suspected that was the reason.